### DISCUSSION TITLE
No internet connectivity

### DISCUSSION THREAD

**Discussion by PANiCnz** [wg-easy/wg-easy#162] [Q&A] [answered]
I've got wg-easy installed and it appears to be running successfully, but when my iPhone connects it doesn't have an internet connection and I can access services on the local network. Any tips for getting internet connectivity?

**Comment by domigi** [comment]
Did you have any success in the meantime?

**Comment by leporel** [comment]
i can ping 1.1.1.1 and 8.8.8.8 from client but cant open any website 


<details>
  <summary>also docker spam new connections on my router </summary>
![image](https://user-images.githubusercontent.com/17878203/157924972-1586a696-c1aa-42df-a3bb-b4ef8a22e496.png)
</details>

PS: worked on Bridge, but when using macvlan - websites outside LAN not access

**Comment by pheiduck** [comment, marked as answer]
@PANiCnz please try out the latest image: `ghcr.io/wg-easy/wg-easy:latest`
